tests/function_libs/torch_lib/ops_test_data.py

```diff
@@ -438,6 +438,34 @@ def _where_input_wrangler(
     return args, kwargs
 
 
+def _unique_unsorted_xfail_matcher(
```

Comment:
@justinchuby I'm not sure what the preferred behavior is here. Should we match `torch.unique` and ignore the `sorted` argument (i.e., always sort in `aten_unique`) or respect the argument and deviate in accordance with this matcher?

Reply:
I wonder if the argument leads to different behavior in cuda/cpu etc? I assume sorted=False means it _can_ be sorted, but it doesn't need to be; and there are some potential performance gain by turning it off. If that's the interpretation I would keep the argument. Otherwise ignoring the argument and matching behavior would also be nice.

Reply:
I am investigating differences in behavior between cuda/cpu and have found at least one already (`unique_dim` on CPU ignores the `return_inverse` and `return_counts` arguments whereas the CUDA impl does not). How should these differences be handled? Can the op registration be conditioned by the device somehow?

Reply:
Matching CUDA for now is preferable. Thanks!